src/generatorprofile.cpp

```diff
@@ -464,7 +464,7 @@ void GeneratorProfile::GeneratorProfileImpl::loadProfile(GeneratorProfile::Profi
 
         // Piecewise statement
 
-        mConditionalOperatorIfString = "#if if (#cond)";
+        mConditionalOperatorIfString = "#if if #cond";
```

Comment:
Do we really want to remove those parentheses? I know that they are not strictly necessary, it's just that I find them easier to read and understand the code. However, if it means that it would break PEP8 compliancy, then I am clearly ok with them being removed.

Reply:
They don't actually break PEP8 compliance.

I thought they did and also didn't like them.